examples/basic_usage/main.tf

```diff
@@ -0,0 +1,6 @@
+module "observe_collection_example" {
```

Comment:
Examples should be able to form the initial basis of a user's configuration. Avoid putting `example` in any resource names. Name things the way you want end users to name them. `module.observe_collection` or `module.observe_google_collection` would be good choices.